rotkehlchen/chain/optimism/modules/velodrome/velodrome_cache.py

```diff
@@ -38,6 +38,8 @@
 class VelodromePoolData(NamedTuple):
     pool_address: ChecksumEvmAddress
     pool_name: str
+    token0: ChecksumEvmAddress
+    token1: ChecksumEvmAddress
```

Comment:
toke0 and token1 made me think that this was a EvmToken. I would add `_address` to the names

Reply:
done